spanner/spanner_snippets/spanner/integration_test.go

```diff
@@ -386,6 +388,23 @@ func runBackupSample(t *testing.T, f backupSampleFunc, dbName, backupID, errMsg
 	return b.String()
 }
 
+func runBackupSampleWithRetry(t *testing.T, f backupSampleFunc, dbName, backupID, errMsg string, maxAttempts int) string {
+	var b bytes.Buffer
+	success := testutil.Retry(t, maxAttempts, time.Minute, func(r *testutil.R) {
+		if err := f(&b, dbName, backupID); err != nil {
```

Comment:
Might want to call `b.Reset` at the start of every retry attempt.

Reply:
Good point, done.